libraries/pubcidEids/pubcidEids.js

```diff
@@ -0,0 +1,6 @@
+export const PUBCID_EIDS = {
+  'pubcid': {
+    source: 'pubcid.org',
+    atype: 1
+  }
+}
```

Comment:
We need a test that shows that it works for whatever we expose. Also, this code must support the `ext` field. We set it to `ext: { provider: LI_PROVIDER_DOMAIN }` in our user id module.